### PR TITLE
feat(Stripe Trigger Node): Add Stripe webhook descriptions based on the workflow ID and name

### PR DESCRIPTION
**Pull request by JavierCane**
## Summary

Right now n8n creates the needed Stripe Webhooks while adding Stripe Trigger nodes without specifying a description for that Stripe Webhook.

That is, you can end up having a list of Stripe Webhooks which do not reveal any kind of contextual information. In case of receiving an errors spike alert from Stripe, which demands a quick response, you do not have any more information rather than the webhook URL where it points to:

With this PR n8n will start adding the following description to these webhooks:

```javascript
`Created by n8n for workflow ID $workflowId: $workflowName`;
```

This allows to have a better idea of the implications of an error spike in webhook calls, or just knowing why you have _N_ declared Stripe Webhooks and where they come from.

Take into account that, even if you manually add the Stripe Webhook description from the Stripe Workbench, it will be removed while updating n8n because it de-register the webhooks and create them again. So this is the only approach I can think of in order to maintain the webhook descriptions in place 😊

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md))
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included.
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

**Comment by Joffcom** [comment]
Hey @JavierCane 

I really like this idea, the version of the node doesn't need increasing where you have done it so that can be reverted.

As workflow names can contain any character including emojis does this cause any issues on the Stripe side? It may be safer to just use the workflow ID which solves the issue, what do you think?

**Comment by JavierCane** [comment]
They do seem to actually support emojis:


If you ask my opinion, I would prefer to provide the workflow name as it will be more semantic than just the ID. However, no strong opinions here. If you prefer so I can remove the title and go ahead with the ID 😊

**Comment by JavierCane** [comment]
Hi, would it make it easy to merge this PR if I remove the workflow name from the Stripe Webhook description?

Thanks!

**Comment by Joffcom** [comment]
Hey @JavierCane,

This is waiting for my next PR review session, I typically look at PRs twice a week. I think I would be more comfortable with just the ID which solves the same problem with less risk.

**Comment by JavierCane** [comment]
No problem then @Joffcom, workflow name removed 🤟

**Comment by Joffcom** [comment]
👍🏻 , I will add this to the review list to test this week.

**Review by Joffcom — APPROVED** [review]

**Comment by JavierCane** [comment]
Hi!

To avoid this falling too far behind `main` and potentially causing merge conflicts, is there anything left I can do before you can merge it?

Thanks!

**Comment by janober** [comment]
Got released with [`n8n@1.54.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.54.0)

**Pull request merged**
